### DISCUSSION TITLE
Fail to build app on Github Actions

### DISCUSSION THREAD

**Discussion by erayerdin** [tauri-apps/tauri#4652] [Q&A]
First of all, [this](https://github.com/erayerdin/taurapp/blob/feat/snap/.github/workflows/build.linux.yaml) is how I configured Linux build for Github Actions.

[This](https://github.com/erayerdin/taurapp/blob/feat/snap/.github/workflows/build.linux.yaml#L29) line right here shows the dynamic build dependencies.

However, I could not get over [this error](https://github.com/erayerdin/taurapp/runs/7290165822?check_suite_focus=true), saying that `javascriptcoregtk-4.0` is missing on CI machine.

The only package for Ubuntu that seems reasonable is `gir1.2-javascriptcoregtk-4.0`, but installing it does not seem to resolve any problem.

The app builds fine on my machine btw.

How do I resolve this?

Thanks in advance.

**Comment by FabianLars** [comment]
you're missing the webkit2gtk dependency (`libwebkit2gtk-4.0-dev` on [ubuntu](https://tauri.app/v1/guides/getting-started/prerequisites#setting-up-linux)). javascriptcore is part of that.
